flake.nix

```diff
@@ -31,19 +39,21 @@
       perSystem = { self', pkgs, ... }:
         let
           # TODO: Change username
-          myUserName = "john";
+          myUserName = "runner";
         in
         {
           legacyPackages.homeConfigurations.${myUserName} =
-            self.nixos-flake.lib.mkHomeConfiguration
+            inputs.self.nixos-flake.lib.mkHomeConfiguration
               pkgs
               ({ pkgs, ... }: {
-                imports = [ self.homeModules.default ];
+                imports = [ inputs.self.homeModules.default ];
```

Comment:
To address [this comment](https://github.com/juspay/nix-dev-home/pull/7/files#r1320630741):

```nix
imports = [ ./home ];
```

Reply:
https://github.com/juspay/nix-dev-home/pull/7/commits/968444174202d146ab598b0d5c215dcae5b810c2